app/templates/gulpfile.babel.js

```diff
@@ -0,0 +1,320 @@
+// Generated on <%= (new Date).toISOString().split('T')[0] %> using <%= pkg.name %> <%= pkg.version %>
+'use strict';
+
+import _ from 'lodash';
+import gulp from 'gulp';
+import path from 'path';
+import gulpLoadPlugins from 'gulp-load-plugins';
+import http from 'http';
+import open from 'open';
+import lazypipe from 'lazypipe';
+import {stream as wiredep} from 'wiredep';
+import nodemon from 'nodemon';
+import runSequence from 'run-sequence';<% if(filters.stylus) { %>
+import nib from 'nib';<% } %>
+
+var plugins = gulpLoadPlugins();
+var config;
+
+var yeoman = {
+    app: require('./bower.json').appPath || 'client/app'
+};
+
+var paths = {
+    client: {
+        scripts: [
+            'client/**/*.<%= scriptExt %>',
+            '!client/bower_components/**/*.js'
+        ],
+        styles: ['client/{app, components}/**/*.<%= styleExt %>'],
```

Comment:
It is not working in gulp styles from components are not being included, using as in script worked
```['client/**/*.<%= styleExt %>' ,'!client/bower_components/**/*.*]```